docs/topics/testing/tools.txt

```diff
@@ -296,55 +317,85 @@ Use the ``django.test.Client`` class to make requests.
         If you set ``secure`` to ``True`` the client will emulate an HTTPS
         request.
 
-    .. method:: Client.head(path, data=None, follow=False, secure=False, **extra)
+        .. versionchanged:: 4.2
+
+            The ``headers`` parameter was added to support specifying headers
+            through a dictionary of names, without the ``HTTP_`` prefix.
+
+    .. method:: Client.head(path, data=None, follow=False, secure=False, *, headers=None, **extra)
 
         Makes a HEAD request on the provided ``path`` and returns a
         ``Response`` object. This method works just like :meth:`Client.get`,
-        including the ``follow``, ``secure`` and ``extra`` arguments, except
-        it does not return a message body.
+        including the ``follow``, ``secure``, ``headers`` and ``extra``
+        arguments, except it does not return a message body.
+
+        .. versionchanged:: 4.2
 
-    .. method:: Client.options(path, data='', content_type='application/octet-stream', follow=False, secure=False, **extra)
+            The ``headers`` parameter was added to support specifying headers
+            through a dictionary of names, without the ``HTTP_`` prefix.
+
+    .. method:: Client.options(path, data='', content_type='application/octet-stream', follow=False, secure=False, *, headers=None, **extra)
 
         Makes an OPTIONS request on the provided ``path`` and returns a
         ``Response`` object. Useful for testing RESTful interfaces.
 
         When ``data`` is provided, it is used as the request body, and
         a ``Content-Type`` header is set to ``content_type``.
 
-        The ``follow``, ``secure`` and ``extra`` arguments act the same as for
-        :meth:`Client.get`.
+        The ``follow``, ``secure``, ``headers`` and ``extra`` arguments act
+        the same as for :meth:`Client.get`.
 
-    .. method:: Client.put(path, data='', content_type='application/octet-stream', follow=False, secure=False, **extra)
+        .. versionchanged:: 4.2
+
+            The ``headers`` parameter was added to support specifying headers
+            through a dictionary of names, without the ``HTTP_`` prefix.
+
+    .. method:: Client.put(path, data='', content_type='application/octet-stream', follow=False, secure=False, *, headers=None, **extra)
 
         Makes a PUT request on the provided ``path`` and returns a
         ``Response`` object. Useful for testing RESTful interfaces.
 
         When ``data`` is provided, it is used as the request body, and
         a ``Content-Type`` header is set to ``content_type``.
 
-        The ``follow``, ``secure`` and ``extra`` arguments act the same as for
-        :meth:`Client.get`.
+        The ``follow``, ``secure``, ``headers`` and ``extra`` arguments act
+        the same as for :meth:`Client.get`.
```

Comment:
Previously these lines inconsistently used the [serial comma](https://en.wikipedia.org/wiki/Serial_comma) - let’s use it consistently 😅

Also we should be consistent about the distinction between [parameters](https://docs.python.org/3.10/glossary.html#term-parameter) and arguments

```suggestion
        The ``follow``, ``secure``, ``headers``, and ``extra`` parameters act
        the same as for :meth:`Client.get`.
```

^ here and below